content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Lambda Function Log Collection Troubleshooting Guide
+kind: documentation
+---
+
+If you don't see logs forwarded from a lambda function in the Log Explorer, see the [Datadog-AWS Log integration][1] to configure your environment. If you still do not see your logs, double-check the following points:
+
+There are some common issues that can come up when configuring the AWS Lambda function to forward logs to Datadog. Below are some troubleshooting steps to try if you are using the Lambda function and your logs aren't reaching Datadog. If you continue to have trouble after following these steps, [contact support][2] for further assistance.
+
+## Are your logs sent to Datadog?
+
+1. Navigate to the [Log Explorer's Live Tail View][3].
+2. In the Search Bar, use a filter to limit the Live Tail View to just the logs coming from your lambda function. Some common search queries are:
+    * By source: the source is often set to `source:aws` or `source:cloudwatch` but you can find other possible sources in the `parse_event_source` function in the [Lambda function][4]. 
+    * By forwarder name: the Lambda function adds a forwardername tag to all the logs it forwards. You can filter on this tag by searching for `forwardername:*` or `forwardername:<function_name>`.
+3. If you see your logs in the Live Tail View but they are not reaching the Log Explorer, this could be due to one of the following reasons:
+    * Logs are too far in the past: Datadog only accepts logs with a timestamp older than 6 hours in the past or 1h in the future.
+    * If your index has any [exclusion filters][5] set up, they may be filtering out your logs.
+
```

Comment:
I would add a transition sentence here, like:

 If you don't see the logs in the livetail, the logs are not reaching Datadog so let's investigate the most common configuration issues.

Or something similar